<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Synthesiser</class>
 <widget class="QMainWindow" name="Synthesiser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lbBeat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbHold">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Hold previous beat</string>
    </property>
   </widget>
   <widget class="QDial" name="dNote">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>68</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbNote">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QDial" name="dVolume">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="dAttack">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="dRelease">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="dSustain">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="blPrevious">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Backwards</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btNext">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Forward</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbOctave">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
